examples/solvers/pdhg_denoising_ROF_algorithm_comparison.py

```diff
@@ -0,0 +1,124 @@
+"""Total variation denoising using the primal-dual hybrid gradient algorithm.
```

Comment:
Not quite clear what this function adds? I thought we had something similar before.

Reply:
Sounds good!